restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/faultTolerance/ActorFaultToleranceTest.java

```diff
@@ -0,0 +1,214 @@
+package org.restcomm.connect.testsuite.faultTolerance;
+
+import akka.actor.ActorPath;
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.OneForOneStrategy;
+import akka.actor.Props;
+import akka.actor.SupervisorStrategy;
+import akka.actor.UntypedActor;
+import akka.event.Logging;
+import akka.event.LoggingAdapter;
+import akka.japi.Function;
+import akka.testkit.JavaTestKit;
+import akka.testkit.TestProbe;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.restcomm.connect.commons.faulttolerance.RestcommSupervisor;
+import scala.concurrent.Await;
+import scala.concurrent.duration.Duration;
+
+import java.util.concurrent.TimeUnit;
+
+import static akka.actor.SupervisorStrategy.escalate;
+import static akka.actor.SupervisorStrategy.resume;
+import static akka.actor.SupervisorStrategy.stop;
+import static akka.pattern.Patterns.ask;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Created by gvagenas on 22/02/2017.
+ */
+public class ActorFaultToleranceTest {
```

Comment:
can we add tests for sending following messages to `RestcommSupervisor` :
- Props (test create actor)
- Terminated (test removal)
- StopChild (test child removal)